ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/OnDisk.hs

```diff
@@ -317,13 +324,18 @@ trimSnapshots tracer hasFS DiskPolicy{..} = do
 -------------------------------------------------------------------------------}
 
 -- | On disk snapshots are numbered monotonically
-newtype DiskSnapshot = DiskSnapshot Int
+data DiskSnapshot = DiskSnapshot {
+    dsSlotNo :: !(WithOrigin SlotNo)
```

Comment:
There's no point in making a snapshot of the ledger state at genesis, we are given this state at startup.